ui/app/src/modules/Preview/Tools/ComponentsPanel.tsx

```diff
@@ -162,6 +169,10 @@ export function ComponentsPanelUI(props) {
 
   const onMenuOptionClicked = () => setMenuContext(null);
 
+  const onListWelcomingReceptaclesClick = () => {
```

Comment:
let's just name receptacles instead of welcoming receptacles all across

e.g. `onListReceptacles`